### PR TITLE
- Added support TTL

### PR DESCRIPTION
**Pull request by IBazylchuk**
- Added support TTL
- Added support tables without timestamps
- Added increment/decrement methods

### CODE REVIEW

**Comment by andrykonchin** [comment]
Is this PR ready for review?

**Comment by IBazylchuk** [comment]
@andrykonchin Yes, probably require some refactors for same style

**Comment by andrykonchin** [comment]
Could you make a separate commit for each feature and add some examples with proposed DSL, e.g. it isn't clear what you name as TTL.

Also new features require new tests.

**Comment by pboling** [comment]
@IBazylchuk Please also rebase on latest master!

**Comment by noelrocha** [comment]
no ttl support yet?

File: dynamoid.gemspec [hunk 1]
```diff
@@ -49,7 +49,7 @@ Gem::Specification.new do |spec|
   # end
   spec.add_runtime_dependency(%q<activemodel>, [">= 4"])
   spec.add_development_dependency(%q<activesupport>, [">= 4"])
-  spec.add_runtime_dependency(%q<aws-sdk-resources>, ["~> 2"])
+  spec.add_runtime_dependency 'aws-sdk-dynamodb', '~> 1'
```
Comment (pboling) [inline dynamoid.gemspec]: What is this change?

**Comment by andrykonchin** [comment]
All of the mentioned features are implemented already.

**Pull request closed without merging**